templates/zerver/create_realm.html

```diff
@@ -0,0 +1,47 @@
+{% extends "zerver/portico_signup.html" %}
+{# Home page for not logged-in users. #}
+
+{# This is where we pitch the app and solicit signups. #}
+
+{% block portico_content %}
+<script type="text/javascript">
+$(function () {
+    autofocus('#email');
+});
+</script>
+
+<div class="app register-page">
+  <div class="app-main register-page-container">
+
+        <div class="register-form">
+            <p class="lead">
+              <div class="register-page-header">{{ _("Let's get started") }}…</div>
+            </p>
+            <form class="form-inline" id="send_confirm" name="email_form"
+                  action="{{ current_url() }}" method="post">
+                {{ csrf_input }}
+                <input type="text" class="email required" placeholder="{{ _("Enter your work email address") }}"
+                       id="email" name="email"/>
+                <input type="submit" class="btn btn-primary btn-large register-button" value="{{ _("Create team") }}"/>
+            </form>
+            <div id="errors"></div>
+            {% if form.email.errors %}
+                {% for error in form.email.errors %}
+                    <div class="alert alert-error">{{ error }}</div>
+                {% endfor %}
+            {% endif %}
+
+            <div class="alert alert-pitch" id="company-email">{% trans %}Please use your
+            company email address to sign up. Otherwise, we won’t be able to
+            connect you with the members of your organization.{% endtrans %}</div>
```

Comment:
I think this notice doesn't apply to the "create a new team" page, since they are the first member of the organization; maybe just delete it?  (more leftover code from my copy-pasted from user registration prototype)